addons/mail/static/src/owl/components/chatter/chatter_component_wrapper.js

```diff
@@ -0,0 +1,17 @@
+odoo.define('mail.component.ChatterComponentWrapper', function (require) {
+'use strict';
+
+const { ComponentWrapper } = require('web.OwlCompatibility');
+const { tags: { xml } } = owl;
+
+class ChatterComponentWrapper extends ComponentWrapper {}
+
+ChatterComponentWrapper.template = xml`<div>
+    <t t-if="props.chatterLocalId">
+        <t t-component="Component" t-props="props" t-ref="component"/>
+    </t>
+</div>`;
+
```

Comment:
Also define `ChatterComponentWrapper.props` (with Object.assign together with template as we do for other components)